hwpm: get_user_pages() lost the vmas argument in 6.5

See linux commit 54d020692b34 ("mm/gup: remove unused vmas parameter
from get_user_pages()").

diff --git a/drivers/tegra/hwpm/os/linux/mem_mgmt_utils.c b/drivers/tegra/hwpm/os/linux/mem_mgmt_utils.c
index d11132e..84de25b 100644
--- a/drivers/tegra/hwpm/os/linux/mem_mgmt_utils.c
+++ b/drivers/tegra/hwpm/os/linux/mem_mgmt_utils.c
@@ -455,7 +455,7 @@ int tegra_hwpm_map_update_allowlist(struct tegra_soc_hwpm *hwpm,
 		hwpm->alist_map->num_pages, 0, hwpm->alist_map->pages);
 #else
 	pinned_pages = get_user_pages(user_va & PAGE_MASK,
-		hwpm->alist_map->num_pages, 0, hwpm->alist_map->pages, NULL);
+		hwpm->alist_map->num_pages, 0, hwpm->alist_map->pages);
 #endif
 	if (pinned_pages != hwpm->alist_map->num_pages) {
 		tegra_hwpm_err(hwpm, "Requested %llu pages / Got %ld pages",
